### PR TITLE
added install instructions on linux

### PR DESCRIPTION
**Pull request by marvin-ad-martianum**
as you recommended here as a pull request. Just a suggestion for the readme as i am not 100% sure whats the best way to do this, however the suggestion i made worked on several machines like a charm.

### CODE REVIEW

**Comment by nicolapiccinelli** [comment]
I just saw your request, there are a couple of things we need to discuss:

1. The `configure.sh` was meant for CI/CD so it set the development environment. The test suite is not something you need if you are not a developer of libmpc++.
2. The last part "This will allow you to install the MPC++ library on your system. Using the include..." is clashing a bit with the idea of using CMake to handle dependencies.

**Comment by marvin-ad-martianum** [comment]
ah, interesting with the configure.sh. However, it was extremely convenient for not having to deal with installing the dependencies... 

So what would be the instructions that make sense? just cmake . / make .. / sudo make install? 

or a new configure.sh (without catch2) instead of install the dependencies?

**Comment by nicolapiccinelli** [comment]
We can update the script with a flag to disable the testing suite, some like `configure.sh --disable-test`?

**Comment by marvin-ad-martianum** [comment]
Sounds like a great idea!

I think the instructions i propose are obvious to a programmer, however for mechanical, electrical or robotics engineers (like me) that i expect to be the main final users of such a library its not so simple and an easy few commands someone like me can copy into a terminal to be up and running will be helpful.

File: README.md [hunk 1]
```diff
@@ -53,3 +53,33 @@ You can build the development image with the command below. Note that the image
...
+Install the library with superuser privileges:
+
+    ```
+    sudo make install
+    ```
+
+This will allow you to install the MPC++ library on your system. Using the include 
+
+    '''
+    #include </usr/local/include/mpc/LMPC.hpp>
```
Comment (AndyZe) [inline README.md:82]: This works for me.

```suggestion
    #include <mpc/LMPC.hpp> 
```

File: README.md [hunk 2]
```diff
@@ -53,3 +53,33 @@ You can build the development image with the command below. Note that the image
 ```console
 docker build -t dev_image .
 ```
+
+## System wide usage on an Ubuntu Linux
+
+If your a developer run the `configure.sh` script with superuser privileges:
```
Comment (AndyZe) [inline README.md:59]:
```suggestion
If you're a developer run the `configure.sh` script with superuser privileges:
```

**Pull request merged**